backend/src/federation/client/1_0/FederationClient.ts

```diff
@@ -39,6 +38,9 @@ export async function requestGetPublicKey(
     }
     logger.warn(`requestGetPublicKey processed without response data`)
   } catch (err) {
-    throw new LogError(`Request-Error:`, err)
+    if (err instanceof GraphQLError) {
```

Comment:
duplicate code - see below

Reply:
yes i know. the whole class is duplicated and only for test purposes on handling several API versions.
The focus for this issue is to keep the handshake the publicKey with a reduced loggin output than the very big "gradido error-page". Otherwise our logfiles will cause an overrun of the filesystem quite soon.